### DISCUSSION TITLE
How can I apply styles on labels in radar chart??

### DISCUSSION THREAD

**Discussion by Dongseung** [nhn/tui.chart#730] [Q&A]
How can I apply styles on labels in radar chart??

I want to apply styles on labels in radar chart such as font-size, color and font-family etc...

As radar chart does not have axis. I am having difficult times

레이더 차트의 라벨들에 스타일을 적용시키고 싶은데 axis가 아니기때문에 어려움을 겪고있습니다. 어떻게 적용시킬수 있나요?

ex
![image](https://user-images.githubusercontent.com/75294699/158133769-b9956009-e3dc-47a6-8722-ddcc04dde774.png)
)

**Comment by jwlee1108** [comment]
@Dongseung 

Sorry, We don't support that feature.
